### PR TITLE
Modified Version of Questionnaire Part

### PR DESCRIPTION
**Pull request by kodariewhs**
Modified Version of Questionnaire Part

### CODE REVIEW

**Comment by kodariewhs** [comment]
I tried to redo it by creating a new branch from master , and hopefully this time the codes are alrite:D

File: app/src/main/java/uk/ucl/group21/currentviewapp/Q1ToQ10.java [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 public class Q1ToQ10 extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_q1_to_q10);
+        setContentView(R.layout.activity_q1_to_q10);}
+
+    public void btnNextQ1to10(View view){
+        Intent intent = new Intent(this, Q11ToQ20.class); //opens the Setting Activity
```
Comment (tvaupel) [inline app/src/main/java/uk/ucl/group21/currentviewapp/Q1ToQ10.java]: make sure to change the comments when you copy paste!

File: app/src/main/java/uk/ucl/group21/currentviewapp/Q21ToQ30.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+public class Q21ToQ30 extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_q21_to_q30);}
+
+    public void btnNextQ21to30(View view) {
+        Intent intent = new Intent(this, Q31ToQ37.class); //opens the Setting Activity
```
Comment (tvaupel) [inline app/src/main/java/uk/ucl/group21/currentviewapp/Q21ToQ30.java]: Same here with the comment, its copied and left there.

File: app/src/main/java/uk/ucl/group21/currentviewapp/Q31ToQ37.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+public class Q31ToQ37 extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_q31_to37);}
+
+    public void btnNextQ31to37(View view) {
+        Intent intent = new Intent(this, Q38ToQ44.class); //opens the Setting Activity
```
Comment (tvaupel) [inline app/src/main/java/uk/ucl/group21/currentviewapp/Q31ToQ37.java]: same story again with comment

File: app/src/main/java/uk/ucl/group21/currentviewapp/Q38ToQ44.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+public class Q38ToQ44 extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_q38_to_q44);}
+
+    public void btnNextQ38to44(View view) {
+        Intent intent = new Intent(this, Q45ToQ50.class); //opens the Setting Activity
```
Comment (tvaupel) [inline app/src/main/java/uk/ucl/group21/currentviewapp/Q38ToQ44.java]: again

File: app/src/main/java/uk/ucl/group21/currentviewapp/Q45ToQ50.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import android.view.View;
+
+public class Q45ToQ50 extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_q45_to_q50);}
+
+    public void btnDoneQ45to50(View view) {
```
Comment (tvaupel) [inline app/src/main/java/uk/ucl/group21/currentviewapp/Q45ToQ50.java]: here too

File: app/src/main/java/uk/ucl/group21/currentviewapp/Questionnaire_Overview.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+public class Questionnaire_Overview extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_questionnaire__overview);}
+
+    public void btnBkToDashboard(View view) {
+        Intent intent = new Intent(this, Dashboard.class); //opens the Setting Activity
```
Comment (tvaupel) [inline app/src/main/java/uk/ucl/group21/currentviewapp/Questionnaire_Overview.java]: again

File: app/src/main/res/layout/activity_database_main.xml [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+        android:layout_toEndOf="@+id/textView22"
+        android:layout_toRightOf="@+id/textView22"
+        android:ems="10"
+        android:inputType="textPersonName" />
+
+    <EditText
+        android:id="@+id/editText_DOB"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+
```
Comment (tvaupel) [inline app/src/main/res/layout/activity_database_main.xml]: delete white spaces!

File: app/src/main/res/layout/activity_questionnaire_menu.xml [hunk 8]
```diff
@@ -1,26 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:tools="http://schemas.android.com/tools"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:paddingLeft="@dimen/activity_horizontal_margin"
-        android:paddingRight="@dimen/activity_horizontal_margin"
```
Comment (tvaupel) [inline app/src/main/res/layout/activity_questionnaire_menu.xml]: why make all these deletions? 
just insert your new code and adapt it to the layout

File: app/src/main/res/values-v21/styles.xml [hunk 9]
```diff
@@ -1,4 +1,5 @@
 <resources>
+
```
Comment (tvaupel) [inline app/src/main/res/values-v21/styles.xml]: added a single empty space....what for?

File: app/src/main/res/values/dimens.xml [hunk 10]
```diff
@@ -5,4 +5,5 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="nav_header_vertical_spacing">16dp</dimen>
     <dimen name="nav_header_height">100dp</dimen>
+    <dimen name="fab_margin">16dp</dimen>
```
Comment (tvaupel) [inline app/src/main/res/values/dimens.xml]: Told you in an earlier comment that this wasn't needed!

File: app/src/main/res/values/styles.xml [hunk 11]
```diff
@@ -7,11 +7,14 @@
...
     </style>
+
     <style name="AppTheme.NoActionBar">
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
     </style>
-    <style name="AppTheme.AppBarOverlay" parent="ThemeOverlay.AppCompat.Dark.ActionBar"/>
-    <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Light"/>
+
+    <style name="AppTheme.AppBarOverlay" parent="ThemeOverlay.AppCompat.Dark.ActionBar" />
```
Comment (tvaupel) [inline app/src/main/res/values/styles.xml]: these aren't changes...just format changes....not required

**Pull request closed without merging**